src/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable */
```

Comment:
- [ ] Kindly remove this comment

Reply:
- [ ] Please, consider removing the eslint disabling, as it was suggested by the previous review 👍